account_asset_management_template/__manifest__.py

```diff
@@ -1,6 +1,7 @@
 # Copyright 2023 Onestein (<http://www.onestein.eu>)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
+# pylint: disable=file-not-used
```

Comment:
Undid adding this comment in [[FIX] Pre-commit - Removed unused comment.](https://github.com/onesteinbv/addons-container/pull/109/commits/dc68f06396fbbbba39daa9d1af856c31ea7493c9).